examples/rust/dates/Makefile

```diff
@@ -14,11 +14,12 @@ wasm:
 	cargo wasi build --lib $(RELFLAGS)
 
 .PHONY: test
-test: debug
+test: wasm
```

Comment:
Similar to the above comment, but opposite.  Changing this dependency to "wasm" implies the target will always be built in debug mode because the`RELFLAGS` variable won't be set.

Reply:
hrm. I think I'm going to remove any dependency then. The thing I was trying to refactor is that when it depends on debug, it builds debug even after having just built a release (which is the module I actually want to test).